packages/seacas/libraries/ioss/src/Ioss_DatabaseIO.C

```diff
@@ -1257,6 +1257,91 @@ namespace Ioss {
 
     return {xx.first, yy.first, zz.first, xx.second, yy.second, zz.second};
   }
+
+  std::vector<size_t> DatabaseIO::get_all_block_offset(const std::string& field_name) const
```

Comment:
The function is actually a private member function that is virtual so that ParallelDatabaseIO can inherit and redefine. In theory, it could simply be a templated namespace utility function defined individually for the base class and the inherited class. I figured it was fine as it is since it is a private member function and so not user facing but I can change it such that these classes don't have too many more class functions on them.